logging/event_logger.h

```diff
@@ -182,12 +183,16 @@ class EventLogger {
 
   explicit EventLogger(Logger* logger) : logger_(logger) {}
   EventLoggerStream Log() { return EventLoggerStream(logger_); }
-  EventLoggerStream LogToBuffer(LogBuffer* log_buffer) {
-    return EventLoggerStream(log_buffer);
+  EventLoggerStream LogToBuffer(
+      LogBuffer* log_buffer,
+      const size_t max_log_size = LogBuffer::kDefaultMaxLogSize) {
```

Comment:
Can we avoid default values for function parameters? Either explicitly specify the value at call sites or overload the function.